benchmarks/lucet-microbenchmarks/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+#[macro_use]
+extern crate criterion;
+
+use criterion::Criterion;
+use lucet_runtime::{DlModule, InstanceHandle, Limits, MmapRegion, Module, Region};
+use lucet_wasi_sdk::{CompileOpts, Lucetc};
+use lucetc::{Bindings, LucetcOpts};
+use std::path::Path;
+use std::sync::Arc;
+use tempfile::TempDir;
+
+fn wasi_bindings() -> Bindings {
+    Bindings::from_file("../../lucet-wasi/bindings.json").unwrap()
+}
+
+fn compile_hello<P: AsRef<Path>>(so_file: P) {
+    let wasm_build = Lucetc::new(&["../../lucet-wasi/examples/hello.c"])
```

Comment:
I would rather have a duplicate of this source file inside the lucet-microbenchmarks directory, these sort of cross-repo file paths have created trouble when we've refactored stuff in the past.

Reply:
for sure; this is temporary until I finish the PR